src/geneweaver/api/schemas/apimodels.py

```diff
@@ -35,3 +36,9 @@ class GeneIdMappingAonReq(BaseModel):
 
     source_ids: List[str]
     species: Species
+
+
+class GeneReturn(BaseModel):
+    """Model for gene endpoint return."""
+
+    data: Union[List[Gene], Gene]
```

Comment:
Since the preferred endpoint only returns a single gene, I think this Model should just be `data: List[Gene]`.

Reply:
Related to comment above